src/utilities/popover/Popover.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef } from 'react';
+import { usePopper } from 'react-popper';
```

Comment:
nit: I am personally not against it, I just remember we had this discussions about using third party libraries vs building it ourselves. I am ok with this as it simplifies things tremendously but maybe another discussion to have? I know some people have mentioned it before. Just a thought :)